pydrad/parse/parse.py

```diff
@@ -257,6 +264,48 @@ def _read_amr(self):
                 self._grid_centers[i] = tmp[0]
                 self._grid_widths[i] = tmp[1]
 
+    def _read_trm(self):
+        """
+        Parse the equation terms file
+        """
+        try:
+            with open(self._trm_filename, 'r') as f:
+                lines = f.readlines()
+        except:
+            log.debug(f'{self._trm_filename} not found')
+            return
+        
+        n_elements = 1 + int(len(lines)/4)
+        self._trm_data = np.zeros([n_elements, 3])
+
+        # The files come in sets of 5 rows
+        #   -- Loop coordinate, and at each one:
+        #   -- Terms of mass equation
+        #   -- Terms of momentum equation
+        #   -- Terms of electron energy equation
+        #   -- Terms of hydrogen energy equation
+        # Right now, we only read 3 values from this:
+        #  the electron heating, hydrogen heating,
+        #  and bolometric radiative losses
+        for i in range(len(lines)):
+            if i % 5 == 3 or i % 5 == 4:
+                line = lines[i].strip()
+                line = line.split()
+            if i % 5 == 3:
+                self._trm_data[int(i/4)][0] = float(line[5])
+                self._trm_data[int(i/4)][1] = float(line[6])
+            if i % 5 == 4:
+                self._trm_data[int(i/4)][2] = float(line[5])
```

Comment:
```suggestion
        for i in range(len(lines)):
            j = int(i/4)
            line = lines[i].strip().split()
            # Electron heating and radiative loss terms from the 
            # electron energy equation
            if i % 5 == 3:
                self._trm_data[j, 0] = float(line[5])
                self._trm_data[j, 1] = float(line[6])
            # Hydrogen heating term from the hydrogen energy
            # equation
            if i % 5 == 4:
                self._trm_data[j, 2] = float(line[5])
```

Reply:
Why drop the first if statement?  Presumably doing `line = lines[i].strip().split()` on every iteration is slower than the `if` check.

Reply:
I suppose it could be embedded into each individual if statement, alternatively.

Reply:
Just for the sake of cleaning things up. I didn't think the performance hit would be significant and at some point we do presumably want to parse each term so that initial `if` wouldn't be needed then.

Reply:
Fair enough.  I guess the code is easier to read through when you don't have to disentangle all the conditionals.